todo_app/mongo.py

```diff
@@ -0,0 +1,87 @@
+import os
+from bson.objectid import ObjectId
+import pymongo 
+from dotenv import load_dotenv
+from todo_app.todo_card import TodoCard
+from datetime import datetime
+
+class Mongo:
+    def __init__(self): 
+        self.user_name=os.getenv('USER_NAME')
+        self.password=os.getenv('PASSWORD')
+        self.mongo_url=os.getenv('MONGO_URL')
+        self.default_database=os.getenv('DEFAULT_DATABASE')
+        self.board_name=os.getenv('BOARD_NAME')
+        self.connection_string = "mongodb+srv://" + self.user_name + ":" + self.password + "@" + self.mongo_url + "/" + self.default_database + "?w=majority"
+
+    def getClient(self):
+        return pymongo.MongoClient(self.connection_string)
+
+    def get_databases(self):
+        client = self.getClient()
+        databases = client.list_database_names()
+        for item in databases:
+            print (item)
+
+    def get_cards_for_board(self):
+        client = self.getClient()
+        to_do_app_database = client["To_Do_App"]
+        to_do_items_collection = to_do_app_database['ToDoItems']
+        cards= {}
+        for item in to_do_items_collection.find({"board_name" : self.board_name}):
+            formatted_date = item["date_last_activity"].strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+            card = TodoCard(item["list_name"], item["_id"], item["name"], formatted_date)
+            cards[item["_id"]]=card        
+        return cards
```

Comment:
Why return a dictionary instead of a list?
```suggestion
        cards= []
        for item in to_do_items_collection.find({"board_name" : self.board_name}):
            formatted_date = item["date_last_activity"].strftime("%Y-%m-%dT%H:%M:%S.%fZ")
            card = TodoCard(item["list_name"], item["_id"], item["name"], formatted_date)
            cards.append(card)
        return cards
```

And then you don't even need the `get_todo_items` function - all that does is convert the dictionary into a list. Even if you did need to convert a dictionary to a list of the values the quicker way is just `list(your_dictionary.values())`, rather than a loop.

Reply:
done